src/test/ensemble_mapping_config_tests.cpp

```diff
@@ -72,6 +72,7 @@ TEST_F(PipelineWithInputOutputNameMappedModel, SuccessfullyReferToMappedNamesAnd
 
     // Load models
     auto modelConfig = DUMMY_MODEL_CONFIG;
+    modelConfig.setPluginConfig(plugin_config_t({{"ENABLE_MMAP", "NO"}}));
```

Comment:
Why do we need this now?

Reply:
This test removes model before unloading it. On windows it is impossible. I think it is better option to move ENABLE_MMAP: NO as default for DUMMY_MODEL_CONFIG and other test configs. Unit tests which rely on YES, will have it manually specified.

Reply:
I think we shouldnt change default plugin config to MMAP=NO in tests, because default value is yes in OV. It would mean we unit test something that is not by default in OVMS. Let's disable mmap manually in **only those tests that were designed to remove models during workload**.

Reply:
Please wrap those:
modelConfig.setPluginConfig(plugin_config_t({{"ENABLE_MMAP", "NO"}}));

then into sth named like:
void adjustConfigForWindowsWorkaroundWithRemovingDirectory(config)

It's clear then why it is needed without looking for change history, its clear its windows specific, and nothing specific for that particular test.

Reply:
Adjusted